lib/level/connections.ex

```diff
@@ -102,6 +102,20 @@ defmodule Level.Connections do
     GroupMemberships.get(user, struct(GroupMemberships, args), info)
   end
 
+  @doc """
+  Fetches the current user's membership.
+  """
+  @spec group_membership(Group.t(), map(), authenticated_context()) :: {:ok, GroupUser.t() | nil}
+  def group_membership(%Group{} = group, _args, %{context: %{current_user: user}}) do
+    with {:ok, %{space_user: space_user}} <- Spaces.get_space(user, group.space_id),
+         {:ok, group_user} <- Level.Groups.get_group_membership(group, space_user) do
+      {:ok, group_user}
+    else
+      {:error, _} ->
```

Comment:
I know I'm just a rando here, but this turning an error tuple into an ok tuple seems kind of odd to me. Maybe it's just an intermediate step, but it would be good to maybe drop a comment in here so when you (or someone else) comes back to this later they're not also confused?

Reply:
Fair point. Still reworking the logic a bit here.